torch/csrc/api/include/torch/nn/options/dropout.h

```diff
@@ -7,14 +7,41 @@
 namespace torch {
 namespace nn {
 
-/// Options for `Dropout` and `FeatureDropout`.
+/// Options for `Dropout` module.
 struct TORCH_API DropoutOptions {
-  /* implicit */ DropoutOptions(double rate = 0.5);
-  /// The probability with which a particular component of the input is set to
-  /// zero.
-  /// Changes to this parameter at runtime are effective.
-  TORCH_ARG(double, rate);
+  /* implicit */ DropoutOptions(double p = 0.5);
+
+  /// The probability of an element to be zeroed. Default: 0.5
+  TORCH_ARG(double, p) = 0.5;
+
+  /// can optionally do the operation in-place. Default: False
+  TORCH_ARG(bool, inplace) = false;
+};
+
+/// Options for `Dropout2d` module.
+using Dropout2dOptions = DropoutOptions;
+
+/// Options for `Dropout3d` module.
+using Dropout3dOptions = DropoutOptions;
+
+namespace functional {
+
+/// Options for `Dropout` functional.
+struct TORCH_API DropoutFuncOptions {
+  /// The probability of an element to be zeroed. Default: 0.5
+  TORCH_ARG(double, p) = 0.5;
+
+  /// can optionally do the operation in-place. Default: False
+  TORCH_ARG(bool, inplace) = false;
+
+  TORCH_ARG(bool, training) = false;
```

Comment:
Based on https://github.com/pytorch/pytorch/blob/3fb9bbc99b8e1af286ca84045697a951f5346832/torch/nn/functional.py#L788, it seems that the default value for `training` should be `true`

Reply:
also it would be awesome to move `TORCH_ARG(bool, training) = false;` above `TORCH_ARG(bool, inplace) = false;`, to be consistent with the argument order of the Python functional `dropout` :D